.github/workflows/ci.yml

```diff
@@ -25,12 +25,13 @@ jobs:
           - 5.4
           - 5.3
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - uses: shivammathur/setup-php@v2
         with:
           php-version: ${{ matrix.php }}
           coverage: xdebug
-      - run: sudo apt-get -qq update || true # update package list and ignore temporary network errors
+          ini-file: development
+      - run: sudo apt-get -qq update || true && sudo apt install -f # update package list and ignore temporary network errors
```

Comment:
The `apt` command complains about being used in automated scripts, I suppose the same arguments would also work for `apt-get`?

Reply:
Yes, they do :+1: